src/init.cpp

```diff
@@ -191,6 +197,17 @@ void Shutdown()
     if (pwalletMain)
         pwalletMain->Flush(true);
 #endif
+
+#if ENABLE_ZMQ
+    if (pzmqNotificationInterface)
```

Comment:
Nit: ``if (pzmqNotificationInterface) {`` as sometime in the future this will be the default code formatting (clang cleanup).